pallets/did/Cargo.toml

```diff
@@ -9,28 +9,43 @@ version = "0.24.0"
 [package.metadata.docs.rs]
```

Comment:
Same here. Mainly just re-ordered alphabetically.

Reply:
and the mock feature ;)